lib/index.ts

```diff
@@ -146,3 +186,22 @@ async function getManifestFiles(
     })
     .filter((i) => i.contents !== "");
 }
+
+async function calcHashOfBinFiles(
+  docker: Docker,
+  files: string[],
+): Promise<BinHashFile[]> {
+  const contents = await Promise.all(files.map((f) => docker.catSafeB64(f)));
+
+  return files
+    .map((g, i) => {
+      const sha1 = crypto.createHash('sha1');
```

Comment:
can we use any other modern hash? `sha256` as a minimum (depends what maven central supports)

Reply:
I tend to agree, however I still dont know what mvn central/plugin supports

Reply:
I added an option to define the type of hash to use (and defaults to sha1 in case it is not given)